src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/config/advice/CancelTaskExceptionAdvice.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.config.advice;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import uk.gov.hmcts.reform.wacaseeventhandler.exceptions.CancelTaskException;
+
+import java.time.LocalDateTime;
+
+@ControllerAdvice
+public class CancelTaskExceptionAdvice {
+
+    @ExceptionHandler(CancelTaskException.class)
+    @ResponseBody
+    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
+    ErrorMessage cancelTaskExceptionHandler(Exception ex) {
+        return new ErrorMessage(ex, HttpStatus.INTERNAL_SERVER_ERROR, LocalDateTime.now());
```

Comment:
For this one, have we checked the format of the time stamp is it RFC3339?  I believe that's the programme standard

Reply:
I use the same date format from another project, don't remember now which one. I have this: `public class ErrorMessage {

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm:ss")
    private final LocalDateTime timestamp;
    private final String error;
    private final int status;
    private final String message;
`